lib/parallel_tests/cli.rb

```diff
@@ -192,7 +192,11 @@ def execute_shell_command_in_parallel(command, num_processes, options)
     def report_time_taken
       start = Time.now
       yield
-      puts "\nTook #{Time.now - start} seconds"
+      elapsed = Time.now - start
+      rounded = elapsed.round
```

Comment:
I'd suggest making this a method like `humanize_time` and then add a few tests to make sure it works right with 0 / 1000/ 10000